L1Trigger/Phase2L1ParticleFlow/src/l1-converters/hgcalinputt_ref.cpp

```diff
@@ -0,0 +1,21 @@
+#include "L1Trigger/Phase2L1ParticleFlow/interface/l1-converters/hgcalinput_ref.h"
+
+l1ct::HgcalClusterDecoderEmulator::~HgcalClusterDecoderEmulator() {}
+
+l1ct::HadCaloObjEmu l1ct::HgcalClusterDecoderEmulator::decode(const ap_uint<256> &in) const {
+  ap_uint<14> w_pt = in(13, 0);
+  ap_uint<14> w_empt = in(27, 14);
+  ap_int<9> w_eta = in(72, 64);
+  ap_int<9> w_phi = in(81, 73);
+  ap_uint<10> w_qual = in(115, 106);
```

Comment:
Again some constants definition may help readability.

Reply:
These are just numbers copy-pasted from a human-written pdf document that define which bits encode what in the dataformat, and hopefully they don't change too much.
Hopefully in the future we get proper dataformats for this stuff, and then we won't need to have the numbers in the code (or at least not in this code)